runtime/doc/farsi.txt

```diff
@@ -61,15 +61,14 @@ directory:
 
    +  far-a01.pcf    X Windows fonts for Unix including Linux systems
    +  far-a01.bf     X Windows fonts for SunOS
-   +  far-a01.f16    a screen fonts for Unix including Linux systems
-   +  far-a01.fon    a monospaced fonts for Windows NT/95/98
-   +  far-a01.com    a screen fonts for DOS
+   +  far-a01.f16    screen fonts for Unix including Linux systems
+   +  far-a01.fon    monospaced fonts for Windows
```

Comment:
This whole `Farsi Fonts` section can be removed, as the fonts referenced above don't seem to exist in the Neovim repo.